inginious/frontend/templates/course_admin/task_list.html

```diff
@@ -113,4 +118,48 @@ <h5 class="modal-title" id="taskDispenserModalLabel">{{_("Switch task dispenser"
         </button>
     </div>
 </div>
+
+<script>
+	const tasksWithErrors = document.getElementsByClassName("bg-danger")
+
+	for(let task of tasksWithErrors){
+		document.getElementById("errorLogContainer").innerHTML += "<br>&nbsp &nbsp &nbsp<a href='#" + task.id + "' style='color: white; text-decoration: underline'>" + task.innerText + "</a><br>"
```

Comment:
Avoid writing HTML code by hand in code, jQuery has the adequate models to build HTML objects.